geode-gfsh/src/integrationTest/java/org/apache/geode/codeAnalysis/AnalyzeGfshSerializablesIntegrationTest.java

```diff
@@ -12,17 +12,23 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-
 package org.apache.geode.codeAnalysis;
 
 import org.junit.experimental.categories.Category;
 
+import org.apache.geode.gfsh.internal.management.GfshSanctionedSerializablesService;
 import org.apache.geode.test.junit.categories.SerializationTest;
 
-@Category({SerializationTest.class})
-public class AnalyzeManagementSerializablesJUnitTest extends AnalyzeSerializablesJUnitTestBase {
+@Category(SerializationTest.class)
+public class AnalyzeGfshSerializablesIntegrationTest extends AnalyzeSerializablesJUnitTestBase {
```

Comment:
Note: github diff is confused... the code is correct.

AnalyzeGfshSerializablesIntegrationTest is an unrelated new test in geode-gfsh.

AnalyzeManagementSerializablesJUnitTest was renamed to AnalyzeManagementSerializablesIntegrationTest which still exists in geode-management.